src/ui/component/publishPrice/view.jsx

```diff
@@ -13,40 +14,44 @@ function PublishPrice(props: Props) {
   const { contentIsFree, fee, updatePublishForm, disabled } = props;
 
   return (
-    <section className="card card--section">
-      <FormField
-        type="radio"
-        name="content_free"
-        label={__('Free')}
-        checked={contentIsFree}
-        disabled={disabled}
-        onChange={() => updatePublishForm({ contentIsFree: true })}
-      />
+    <Card
+      actions={
+        <React.Fragment>
+          <FormField
+            type="radio"
+            name="content_free"
+            label={__('Free')}
+            checked={contentIsFree}
+            disabled={disabled}
+            onChange={() => updatePublishForm({ contentIsFree: true })}
+          />
 
-      <FormField
-        type="radio"
-        name="content_cost"
-        label={__('Add a price to this file')}
-        checked={!contentIsFree}
-        disabled={disabled}
-        onChange={() => updatePublishForm({ contentIsFree: false })}
-      />
-      {!contentIsFree && (
-        <FormFieldPrice
-          name="content_cost_amount"
-          min="0"
-          price={fee}
-          onChange={newFee => updatePublishForm({ fee: newFee })}
-        />
-      )}
-      {fee && fee.currency !== 'LBC' && (
-        <p className="form-field__help">
-          {__(
-            'All content fees are charged in LBC. For non-LBC payment methods, the number of credits charged will be adjusted based on the value of LBRY credits at the time of purchase.'
+          <FormField
+            type="radio"
+            name="content_cost"
+            label={__('Add a price to this file')}
+            checked={!contentIsFree}
+            disabled={disabled}
+            onChange={() => updatePublishForm({ contentIsFree: false })}
+          />
+          {!contentIsFree && (
+            <FormFieldPrice
+              name="content_cost_amount"
+              min="0"
```

Comment:
out of scope but why is `min` a string :stuck_out_tongue:

Reply:
It shouldn't be